src/index.js

```diff
@@ -88,7 +88,7 @@ export function syncReduxAndRouter(history, store, selectRouterState = SELECT_ST
   const unsubscribeHistory = history.listen(location => {
     const route = {
       path: createPath(location),
-      state: location.state
+      state: location.state === null ? undefined : location.state
```

Comment:
Ah, maybe we should change the tests instead? Nice to be consistent with `history`, plus the code is simpler.